src/Menu/StyledMenuItem.js

```diff
@@ -34,7 +35,6 @@ const selectedStyles = ({ selected, theme }) =>
   css`
     color: ${theme.colors.text.dark};
     font-weight: ${theme.fontWeights.bold};
-    padding-right: ${theme.space.smaller};
```

Comment:
this looked bad with secondary text, taking overflows into account, etc. it can be put back in easily enough, but if so, the looks-like-shit-while-selected issue should be resolved as a separate story.